geth-poa/genesis.json

```diff
@@ -20,7 +20,7 @@
       }
     },
     "nonce": "0x0",
-    "timestamp": "0x6546df6d",
+    "timestamp": "0x18E0A4E0D22",
```

Comment:
0x6546df6d * 1000 = `0x18b9cd8c1c8`, changing this genesis starting timestamp shouldn't matter tho right

Reply:
Yeah it doesn't matter